lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -18,6 +18,11 @@ table.ccx-schedule {
     th, td {
         padding: 10px;
     }
+    td.no-link {
+        font-size: 0.8125em;
```

Comment:
Please change to px or rem units (see https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#units-of-measurement).